src/main/resources/static/cas3-domain-events-api.yml

```diff
@@ -551,6 +551,21 @@ components:
         - personReference
         - applicationId
         - applicationUrl
+    CAS3AssessmentUpdatedEvent:
+      allOf:
+        - $ref: '#/components/schemas/CAS3Event'
+        - type: object
+          properties:
+            fieldName:
+              type: string
+            updatedFrom:
+              type: string
+            updatedTo:
+              type: string
```

Comment:
would it be worth making this into an array of updated fields, even if only one is being updated at the moment? If you ever need to support multiple fields in the future it'll be awkward when deserializing the JOSN as you need to consider the 'old' schema. You'll need to deal with schema version control which we have an approach for but best avoided if possible. see https://github.com/ministryofjustice/hmpps-approved-premises-api/blob/main/doc/how-to/modifying_domain_event_schemas.md